src/modules/workspace/abtest/finish.ts

```diff
@@ -39,11 +35,13 @@ const promptWorkspaceToFinishABTest = () =>
 export default async () => {
   await installedABTester()
   const workspace = await promptWorkspaceToFinishABTest()
-  await promptContinue(workspace)
-  log.info('Finishing A/B tests')
-  log.info(`Latest results:`)
-  await abTestStatus()
-  await abtester.finish(workspace)
-  log.info(`A/B testing with workspace ${chalk.blue(workspace)} is now finished`)
-  log.info(`No traffic currently directed to ${chalk.blue(workspace)}`)
+  const promptAnswer = await promptContinue(workspace)
```

Comment:
_nit_:
```
if (!promptAnswer) {
  return
}
```

Reply:
i think this way we can remove some indentation haha

Reply:
Again, lint didn't allowed me.